djangoProject/users/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import RegisterView, LoginView, home
+
+urlpatterns = [
+    path('', home, name='home'),
+    path('register/', RegisterView.as_view(), name='register'),
+    path('login/', LoginView.as_view(), name='login'),
+]
```

Comment:
I did not find this functionality, please check carefully.

from django.urls import path, include

urlpatterns = [
    path('api/', include('users.urls')),
    # other paths...
]

Reply:
If I do that I will get RecursionError